running-coreos/cloud-providers/vultr/index.md

```diff
@@ -11,10 +11,27 @@ weight: 10
 These instructions will walk you through running a single CoreOS node. This guide assumes:
 
 * You have an account at [Vultr.com](https://www.vultr.com).
-* The location of your iPXE script (referenced later in the guide) is located at ```http://example.com/script.txt```
 * You have a public + private key combination generated. Here's a helpful guide if you need to generate these keys: [How to set up SSH keys](https://help.github.com/articles/generating-ssh-keys).
 
-The simplest option to boot up CoreOS is to load a script that contains the series of commands you'd otherwise need to manually type at the command line. This script needs to be publicly accessible (host this file on your own server). Save this script as a text file (.txt extension).
+The simplest option to boot up CoreOS is to select the "CoreOS Stable" operating system from Vultr's default offerings. However, most deployments require a custom `cloud-config`, which can only be achieved in Vultr with an iPXE script. The remainder of this article describes this process.
+
+First, you'll need to make two files available at public URL's -- your `cloud-config` file and the following shell script *(be sure to replace the cloud-config URL)*:
```

Comment:
Agreed. Speaking of notes, I was wondering if the following note could be purged:

>Note that the `$private_ipv4` and `$public_ipv4` variables are only supported on Vultr if you have the 'cloud-config-url' option set on your kernel command line. Without this option, you will need to hard code these values into your `cloud-config` file.

But I left it in since I'm not quite sure what it's talking about. (Why would one ever *not* set the 'cloud-config-url' on their kernel command?)